spinalcordtoolbox/scripts/sct_compute_snr.py

```diff
@@ -84,6 +84,13 @@ def get_parser():
         default=1,
         # Values [0, 1, 2] map to logging levels [WARNING, INFO, DEBUG], but are also used as "if verbose == #" in API
         help="Verbosity. 0: Display only errors/warnings, 1: Errors/warnings + info messages, 2: Debug mode")
+    optional.add_argument(
+        '-o',
+        metavar=Metavar.str,
+        type=str,
+        default=None,
+        help="File name where to write the computed SNR."
```

Comment:
Just a small grammar nitpick...

```suggestion
        help="File name to write the computed SNR to."
```